VERSION

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Do we need the "v", i.e. v0.0.1, as I did in the go-mod-registry or doesn't matter?

Reply:
@lenny-intel Yeah, I had this same question. I will add it, and I suspect we'll need to add to edgex-go as well.